### DISCUSSION TITLE
What to put in as Terraria Version for 1.4.2.2

### DISCUSSION THREAD

**Discussion by MrMysterius** [popstarfreas/Dimensions#81] [General]
I have two TShock servers running on `1.4.2.2` and my client running on the same version, but I can't switch to another server on a server with Dimension command, since my version miss matches. Now I wanted to know what I have to put in `currentTerrariaVersion` in the config to prevent that from happening since it says that that is the version it uses for switching?! Currently, there is `currentTerrariaVersion: 234` and the 234 doesn't make sense for me.

**Comment by tlworks** [comment]
You change 234 to 236, the latest Terraria version can be found in the client config in your steamapps folder.

**Reply by MrMysterius** [reply]
Thanks was 237 already for me but thank you
